src/main/java/ch/njol/skript/effects/EffZombify.java

```diff
@@ -0,0 +1,90 @@
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.SyntaxStringBuilder;
+import ch.njol.skript.util.Timespan;
+import ch.njol.skript.util.Timespan.TimePeriod;
+import ch.njol.util.Kleenean;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.entity.Villager;
+import org.bukkit.entity.ZombieVillager;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Zombify Villager")
+@Description({
+	"Turn a villager into a zombie villager. Cure a zombie villager immediately or after specified amount of time.",
+	"This effect removes the old entity and creates a new entity. Using a variable with an entity stored in it, will not change the variable to the new entity."
```

Comment:
I think we should handle this. zombify() returns the new entity so we can just change the input expression if it accepts it.